tests/e2etest/mock_data_aggregate.rs

```diff
@@ -0,0 +1,376 @@
+    string_endpoint: String,
+    binaryblob_endpoint: Vec<u8>,
+    datetime_endpoint: DateTime<Utc>,
+    doublearray_endpoint: Vec<f64>,
+    integerarray_endpoint: Vec<i32>,
+    booleanarray_endpoint: Vec<bool>,
+    longintegerarray_endpoint: Vec<i64>,
+    stringarray_endpoint: Vec<String>,
+    binaryblobarray_endpoint: Vec<Vec<u8>>,
+    datetimearray_endpoint: Vec<DateTime<Utc>>,
+}
+
+pub struct MockDataAggregate {
+    device_to_server: TestAggregateData,
+    server_to_device: HashMap<String, AstarteType>,
+}
+
+impl MockDataAggregate {
+    /// Initialize a new instance for the MockDataAggregate struct.
+    ///
+    /// Fills the data structs with predefined data.
+    pub fn init() -> Self {
+        let device_to_server = TestAggregateData {
+            double_endpoint: 4.34,
+            integer_endpoint: 1,
+            boolean_endpoint: true,
+            longinteger_endpoint: 45543543534,
+            string_endpoint: "Hello".to_string(),
+            binaryblob_endpoint: base64::engine::general_purpose::STANDARD
+                .decode("aGVsbG8=")
+                .unwrap(),
+            datetime_endpoint: DateTime::<Utc>::from(
+                DateTime::parse_from_rfc3339("2021-09-29T17:46:48.000Z").unwrap(),
+            ),
+            doublearray_endpoint: Vec::from([43.5, 10.5, 11.9]),
+            integerarray_endpoint: Vec::from([-4, 123, -2222, 30]),
+            booleanarray_endpoint: Vec::from([true, false]),
+            longintegerarray_endpoint: Vec::from([53267895478, 53267895428, 53267895118]),
+            stringarray_endpoint: Vec::from(["Test ".to_string(), "String".to_string()]),
+            binaryblobarray_endpoint: Vec::from([
+                base64::engine::general_purpose::STANDARD
+                    .decode("aGVsbG8=")
+                    .unwrap(),
+                base64::engine::general_purpose::STANDARD
+                    .decode("aGVsbG8=")
+                    .unwrap(),
+            ]),
+            datetimearray_endpoint: Vec::from([
+                DateTime::<Utc>::from(
+                    DateTime::parse_from_rfc3339("2021-10-23T17:46:48.000Z").unwrap(),
+                ),
+                DateTime::<Utc>::from(
+                    DateTime::parse_from_rfc3339("2021-11-11T17:46:48.000Z").unwrap(),
+                ),
+            ]),
+        };
+        let server_to_device = utils::initialize_hash_map(
+            Some(("double_endpoint".to_string(), 95.8)),
+            Some(("integer_endpoint".to_string(), 20)),
+            Some(("boolean_endpoint".to_string(), true)),
+            Some(("longinteger_endpoint".to_string(), 45993543534)),
+            Some(("string_endpoint".to_string(), "string for test".to_string())),
+            // Some((
+            //     "binaryblob_endpoint".to_string(),
+            //     base64::engine::general_purpose::STANDARD
+            //         .decode("d29ybGQ=")
+            //         .unwrap(),
+            // )),
```

Comment:
Please do not commit commented code. Leave a TODO comment to make it evident something should be there.

Reply:
Let's do that I open an issue regarding those missing tests.